include/gridtools/storage/adapter/nanobind_adapter.hpp

```diff
@@ -97,10 +98,22 @@ namespace gridtools {
                 .template set<property::lower_bounds>(gridtools::array<integral_constant<std::size_t, 0>, ndim>())
                 .template set<property::upper_bounds>(shape);
         }
+
+        template <class T,
+            array_size_t... Sizes,
+            class... Args,
+            class Strides = fully_dynamic_strides<sizeof...(Sizes)>,
+            class StridesKind = sid::unknown_kind>
+        auto as_const_sid(nanobind::ndarray<T, nanobind::shape<Sizes...>, Args...> ndarray,
+            Strides stride_spec_ = {},
+            StridesKind = {}) {
+            return sid::add_const(std::true_type{}, as_sid(ndarray, stride_spec_, StridesKind{}));
```

Comment:
```suggestion
            StridesKind strides_kind = {}) {
            return sid::as_const(as_sid(ndarray, stride_spec_, strides_kind));
```
`StridesKind` does not need to be default constructible, so maybe it’s better to pass the given value instead of creating a new one. Further, `sid::as_const` instead of `sid::add_const` should be good enough.